src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -53,13 +59,37 @@ public void setShowTime(LocalDateTime showTime) {
         this.showTime = showTime;
     }
 
+    public MovieSession setShowTime(int year, int month, int day, int hour, int minute) {
+        this.showTime = showTime.of(year, month, day, hour, minute);
```

Comment:
In the `setShowTime` method, use `LocalDateTime.of(year, month, day, hour, minute)` instead of `showTime.of(...)` to correctly create a `LocalDateTime` instance.